ci/publish-book.sh

```diff
@@ -16,22 +16,41 @@ echo --- create book repo
   git commit -m "${CI_COMMIT:-local}"
 )
 
-eval "$(ci/channel-info.sh)"
-# Only publish the book from the edge and beta channels for now.
-case $CHANNEL in
-edge)
-  repo=git@github.com:solana-labs/book-edge.git
-  ;;
-beta)
-  repo=git@github.com:solana-labs/book.git
-  ;;
-*)
- echo "--- publish skipped"
- exit 0
- ;;
-esac
+if [[ "$MANUAL_PUBLISH" = true ]]; then
```

Comment:
Could MANUAL_PUBLISH instead be the exact tag we want to publish in `book.git`?

Reply:
I suppose so, but would we ever want to publish an update to the book (clarifications, extra steps, etc) that wasn't paired with a release?

Reply:
Nope. But I'd feel better about feeding automation a tag rather than a commit sha1.

If TAG=v0.16.4 is set the env, that means publish "book".  Else use the existing channel selection logic to publish -edge or -beta (or nothing if there's no matching channel)?

Reply:
@mvines Okay should be all set.  Options were to check out the book source for the tag within the job, or trigger a new job on that tag...I hope opting for the former isn't too sacrilegious.